test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7003,6 +7012,15 @@ def forward(self, x):
         x = torch.tensor([[1, 2, float("inf")], [2, float("nan"), float("inf")]])
         self.run_test(M(), (x, ))
 
+    @skipIfUnsupportedMinOpsetVersion(10)  # ONNX IsInf op is added in opset 10.
```

Comment:
No need for the comment after the skip decorator.

Reply:
done